.github/workflows/release.yaml

```diff
@@ -0,0 +1,63 @@
+on:
+  workflow_dispatch:
+    branches: main
+
+jobs:
+  release:
+    name: Perform release
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git config --global user.email github-actions@github.com
+          git config --global user.name github-actions
+      - name: Cocogitto release
```

Comment:
A release is already created on tag here: https://github.com/JacquesDurand/php-documentation-generator/pull/37

Reply:
It is not the same kind of "release"